Utilities/macrecovery/macrecovery.py

```diff
@@ -195,7 +195,7 @@ def get_image_info(session, bid, mlb=MLB_ZERO, diag=False, os_type='default', ci
     return info
 
 
-def save_image(url, sess, filename='', directory=''):
+def save_image(url, sess, filename='', directory='com.apple.recovery.boot'):
```

Comment:
I guess this change can be reverted. Otherwise good.

Reply:
Currently testing without the override of the function prototype. `com.apple.recovery.boot` directory gets created and `BaseSystem.dmg` is downloading. A few mins and I'll confirm if `BaseSystem.chunklist` as well as the chunklist verification process completes, and I'll push the revert commit for the function prototype